### ISSUE TITLE
Rule possibly only checked within a block

### ISSUE THREAD

**Issue by ghost** [hudochenkov/postcss-styled-syntax#10]
I conjecture that (some?) rules are only checked within blocks.

For example, I would expect the rule shorthand-property-no-redundant-values to catch on the following, since `padding: 10px;` is the shorter way to write it, but it doesn't flag.

```
const MyComponent = styled.div`
  padding: 10px 10px 10px 10px;
`
```

But the rule does correctly flag this:
```
const MyComponent = styled.div`
  & {
    padding: 10px 10px 10px 10px; 
  }
`
```

**Comment by ghost** [comment]
Hmmm, just upgraded to stlyelint v15, and this is no longer an issue. Thanks for the quick turnaround...
